test/cmdlineTests/optimizer_inliner_dynamic_reference/output

```diff
@@ -156,6 +156,7 @@ sub_0: assembly {
       revert(0x00, 0x24)
         /* "#utility.yul":196:425   */
     tag_22:
+        /* "#utility.yul":236:239   */
```

Comment:
This is an example of the return variable initialization now referring to the return variable name in the source.